src/components/Restoration/RestorationType.vue

```diff
@@ -115,21 +130,5 @@ export default class RestorationType extends Vue {
       this.setRestorationExpiry(null)
     }
   }
-
-  @Watch('$route')
-  private async scrollToInvalidComponent (): Promise<void> {
-    if (this.$route.name === RouteNames.RESTORATION_BUSINESS_NAME) {
```

Comment:
This is done in the parent, consistent with all other step components.

Reply:
This is done in the parent, consistent with all other step components (@JazzarKarim ).